packages/mongodb-memory-server-core/src/util/errors.ts

```diff
@@ -0,0 +1,4 @@
+export class FileNotFoundError {
```

Comment:
```suggestion
export class FileNotFoundError extends Error {
```

shouldnt it extend an normal `Error`?